flink-libraries/flink-table/src/test/scala/org/apache/flink/table/ExpressionReductionTest.scala

```diff
@@ -155,15 +155,15 @@ class ExpressionReductionTest extends TableTestBase {
       "DataSetCalc",
       batchTableNode(0),
       term("select",
-        "13 AS _c0",
+        "CAST(13) AS _c0",
```

Comment:
Yes, because I changed the default nullable to `true`, but the reduced constant is `NOT NULL`, so a `CAST` is here. Do you have any ideas to fix this? 

The default nullable changed to `true` is because `UserDefinedScalarFunctionTest.testResults` and `ArrayTypeTest.testArrayLiterals` fail.